lib/pages/chat_list/chat_list.dart

```diff
@@ -282,12 +285,20 @@ class ChatListController extends State<ChatList>
     return quickCheck;
   }
 
-  List<Room> get filteredRooms => Matrix.of(context)
-      .client
-      .rooms
-      .where(getRoomFilterByActiveFilter(activeFilter))
-      .where(hideBotsRoomFilter)
-      .toList();
+  Future<void> _loadFilteredRooms() async {
```

Comment:
I don't understand why you needed to touch this. It increases complexity, reduces modularity and impacts performance